File: script/scale_1981_2000.m
function dataout=scale_1981_2000(datain,years)
%datain=spi4_3';
%years=1950:2020;
%datain=years;


period=1981:2000;
[~,~,iok]=intersect(period, years);
avg=nanmean(datain(iok),1);
devstd=std(datain(iok),'omitnan');
dataout=(datain-avg)/devstd;


    




     

File: script/scale_1981_2000.R
scale_1981_2000 <- function(datain,years) {
  period=1981:2000
  iok=match(period, years)
  avg=mean(datain[iok],na.rm=TRUE)
  std=sd(datain[iok],na.rm=TRUE)
  dataout=(datain-avg)/std
}